### PR TITLE
elfutils: add v0.191

### PR DESCRIPTION
**Pull request by wdconinc**
This PR adds version 0.191 of elfutils, titled "Bug fixes in C major". News at top of https://sourceware.org/git/?p=elfutils.git;a=blob_plain;f=NEWS;hb=18a015c0b0787ba5acb39801ab7c17dac50f584d, nothing that looks like it would need adapting in the spack package.

Test build (without debuginfod):
```
==> Installed packages
-- linux-ubuntu24.04-skylake / gcc@13.2.0 -----------------------
orv7bze elfutils@0.191~debuginfod+exeprefix+nls build_system=autotools
==> 1 installed package
```

### CODE REVIEW

**Review by alecbcs — APPROVED** [review]
Verified Checksums. Looks good to me. Waiting on final approval from @mwkrentel.

**Review by mwkrentel — APPROVED** [review]
I approve, please merge.

**Pull request merged**
